### PR TITLE
fix: added missing vertexai dependency

### PR DESCRIPTION
**Pull request by GuillermoBlasco**
Upon request to vertexai gemini an error wrapped as a 400 is trhown:
image

This error was generated by dynamic imports. For example here:
https://github.com/BerriAI/litellm/blob/942b5e4145fded85e3795a92c3b934706614ef7f/litellm/llms/vertex_ai.py#L174
image

This makes sense for the client but not for the proxy server.

Adding the suggested dependency fixes the problem.

## Considerations

The docker image increases in size in +100 MB
image

This makes sense for the server proxy, but not for the client, the client should install this as an optional dependency. Regarding this point I did not see a clear way to set this up in poetry consistently with the rest of the pyproject config, so up to you for suggestions or edit.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -39,6 +35,7 @@ streamlit = {version = "^1.29.0", optional = true}
 fastapi-sso = { version = "^0.10.0", optional = true }
 PyJWT = { version = "^2.8.0", optional = true }
 python-multipart = { version = "^0.0.6", optional = true }
+google-cloud-aiplatform = "^1.43.0"
```
Comment (ghost) [inline pyproject.toml:38]: hi @GuillermoBlasco, does this need to be in the client sdk's pyproject.toml? the issue you faced seemed docker env specific

Comment (GuillermoBlasco) [reply pyproject.toml:38]: it is in the proxy. Read in the PR Comment:
image

Comment (ghost) [reply pyproject.toml:38]: Do this `google-cloud-aiplatform = { version = "^1.43.0", optional = true }`

I believe that should fix it

**Comment by ishaan-jaff** [comment]
We've added the dependency to the proxy https://github.com/BerriAI/litellm/blob/bcd62034ed4355bf1449abff6122af872a692ac7/requirements.txt#L16

Can we close this PR @GuillermoBlasco ?

**Comment by ishaan-jaff** [comment]
closing since this is fixed - we've added this to requirements.txt. Feel free to re-open this PR if i'm mistaken @GuillermoBlasco

**Pull request closed without merging**